upgrade/install-1.6.3.php

```diff
@@ -0,0 +1,49 @@
+<?php
+/**
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to a commercial license from SARL 202 ecommence
```

Comment:
202 ecommerce a de not ecommeNce.

please use the licence of the shoppingfeed module class